store/index.js

```diff
@@ -207,15 +278,31 @@ export const actions = {
 			column: p_linkingData.column,
 			category: p_linkingData.category
 		});
-	},
+	},	
 
-	unlinkColumnWithCategory(p_context, p_linkingData) {
+	unlinkColumnFromCategory(p_context, p_linkingData) {
 
 		p_context.commit("removeColumnCategorization", p_linkingData.column);
 	},
 
 	// Annotation page actions
 
+	revertColumnToOriginal(p_context, p_columnName) {
```

Comment:
Could you add a docstring comment to explain what this method does and when it would be called? I assume this is when someone wants to reset an annotation?